support-frontend/assets/components/thankYou/thankYouModuleData.tsx

```diff
@@ -118,14 +113,14 @@ export const getThankYouModuleData = (
 	csrf: CsrfState,
 	isOneOff: boolean,
 	amountIsAboveThreshold: boolean,
+	startDate: string,
```

Comment:
I noticed that we sometimes pass in an empty string here, presumably to represent no `startDate`, which I think has allowed us to tighten the type here to just `string` (before it was optional). I wonder if using empty string is desirable, and whether `null` or `undefined` would be better? To me those more clearly represent the lack of start date. What do you reckon?

Reply:
Previously had this but can see the advantage of `undefined` theoretically not displaying the whole line containing the date at all - a better situation. Also included some stories for the `whatNext` component directly affected. Updated.